go/datas/database_server.go

```diff
@@ -56,7 +57,7 @@ func (s *RemoteDatabaseServer) Run() {
 	d.Chk.NoError(err)
 	s.port, err = strconv.Atoi(port)
 	d.Chk.NoError(err)
-	fmt.Printf("Listening on port %d...\n", s.port)
+	log.Printf("Listening on port %d...\n", s.port)
```

Comment:
This was never protected by the verbose flag, so I'm assuming that it should always be printed. Just making sure it goes through the logger.